### PR TITLE
Remove Gitter links

### PR DESCRIPTION
**Pull request by jsumners**
This PR removes the Gitter links from the readme. People are using it as a "your our only hope @mcollina" channel because 9 times out of 10 he is the only one answering anything. I _occasionally_ answer some things there, but not nearly enough. The community is not helping itself as desired through this avenue. Thus, I am proposing this PR and subsequently closing down the Gitter channel.

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -193,7 +193,6 @@ matters to you.
 
 ## Support
 - [Fastify help](https://github.com/fastify/help)
```
Comment (mcollina) [inline README.md]: Can you add some text before this link? A list of only 1 element is not nice.

**Comment by jsumners** [comment]
> A few other resources would need to be updated:
> 
> https://github.com/fastify/.github/blob/master/.github/ISSUE_TEMPLATE/bug.md
> 
> https://github.com/fastify/.github/blob/master/.github/ISSUE_TEMPLATE/feature.md
> 
> https://github.com/fastify/.github/blob/master/.github/ISSUE_TEMPLATE/question.md
> 
> https://github.com/fastify/.github/blob/master/.github/ISSUE_TEMPLATE/regression.md
> 
> https://github.com/fastify/fastify/blob/master/.github/ISSUE_TEMPLATE.md

Can I get maintainership access to that repo?

File: README.md (hunk 1 above)
Comment (jsumners) [reply README.md]: Updated.

**Comment by mcollina** [comment]
@jsumners this is in this repo to fix: https://github.com/fastify/fastify/blob/master/.github/ISSUE_TEMPLATE.md

**Comment by mcollina** [comment]
@jsumners you should have access to that repo.

**Comment by jsumners** [comment]
@mcollina oops. I was thinking it was the `fastify/.github` repository you were telling me about. Anyway, I have updated the files there (see https://github.com/fastify/.github/pull/2) and removed them here. After this PR is merged, the global versions in `fastify/.github` should be used.

**Pull request merged**